### DISCUSSION TITLE
Development state

### DISCUSSION THREAD

**Discussion by bartez33a** [wrazik/TankBotFight#55] [General]
Hi! Is this project still developed?

**Comment by wrazik** [comment]
Yes, it is! It was long break, but recently I've started with co-workers to add new features during coding-dojos.

**Comment by bartez33a** [comment]
I'm newbie in c++. I have done few small projects on my own. Now I learn c++ and I want to contribute to a project.

**Reply by wrazik** [reply]
Great to hear that! There is list of open tasks: https://github.com/wrazik/TankBotFight/issues . If you are looking for some super-easy task at the beginning - I can also figure out something! What do you say about adding some sound? Lets' say - adding some sound when Tank fires a shot. You can find some sound on your own (e.g. this one? https://freesound.org/people/qubodup/sounds/168707/). This is some tutorial how to play sound with SFML: https://www.sfml-dev.org/tutorials/2.6/audio-sounds.php .

Feel free to ask any questions!

**Reply by bartez33a** [reply]
Thank you very much. 
I will try to do it!

**Comment by bartez33a** [comment]
Hi. 
I added pull request. Should I do something with my code?

**Reply by wrazik** [reply]
No, no, it's great, I just have a super-busy week :(

**Reply by wrazik** [reply]
If you need any help with the rebase, let me know! I'm on CodersSchool Discord :)

**Comment by bartez33a** [comment]
I think that I can handle it. Right now I don't have access to my computer
so I can't compile the code after change and I want to make sure that it
works as expected.

If I have any problem I will ask you, thanks.
